org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -178,6 +178,17 @@ class Translator {
     }
 
     /**
+     * Adding for consistency across hubs
+     */
+    deauthorizeToken(authInfo) {
+        if (!authInfo) {
```

Comment:
The Travis check marked the unused variable as an error - looking now to see why it was an error not warning like the other providers. If we implement deauthorize in the other providers we might need the authInfo, so I'd like to keep the variable being passed in.